pages/blog.mdx

```diff
@@ -1,3 +1,48 @@
-# Blog
+---
+description: The latest updates and releases from the team at Allo.
+type: posts
+---
 
-test
+import { getPagesUnderRoute } from "nextra/context";
+import Link from "next/link";
+
+export function BlogHeader() {
```

Comment:
Could you pull these out into components in the `components/` directory? One per file, so a `components/BlogHeader.tsx` file and a `components/BlogIndex.tsx` file

Reply:
Fixed!